tests/test_transform_no_download.py

```diff
@@ -129,19 +130,20 @@ def subset_result(unittest_adapter, output_dir):
         variables = None
 
     # compare output file if it is geotiff
-    if adapter.get_filetype(unittest_adapter.subsetted_file) != "tif":
+    downloaded_file_type = get_file_type(unittest_adapter.downloaded_file)
+    subsetted_file_type = get_file_type(unittest_adapter.subsetted_file)
```

Comment:
I've only really made the changes necessary to maintain these tests. I do expect, though, that these tests might see a more extensive overhaul in the near future.